.github/pull_request_template.md

```diff
@@ -1,16 +1,37 @@
-<!--- Provide a general summary of your changes in the Title above -->
+# Du oppretter en ny pull request ⭐🤩🎉 
 
-## Description
-<!--- Describe your changes in detail -->
+## Beskrivelse
```

Comment:
Vi trenger kanskje ikke en heading for beskrivelse? Det kan bare være toppen.

Når du oppretter en PR så tar den jo automatisk beskrivelsen fra Git Commit meldingen din. Så er litt kjipt om du så må flytte den ned under rett heading. Istedenfor å bare ha top biten som beskrivelse...